tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,372 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# <center>Computing the count rate of observing HAT-P-11 with the APO 3.5m telescope using ``synphot``</center>"
```

Comment:
I don't see this title rendered when I click "View File". Hmm.

Also, if we really go the notebook route, there needs to be a brief description of what the star is, or maybe simply linking it to https://en.wikipedia.org/wiki/HAT-P-11 .

Reply:
I don't see it either, but it loads just fine from the terminal. I think it's a github thing..

Reply:
My hunch is that the `<center>` tags are confusing GitHub, but that is not important right now.